### PR TITLE
Adding LHCb test Dockerfile.

### PR DESCRIPTION
**Pull request by alexanderrichards**

### CODE REVIEW

**Comment by alexanderrichards** [comment]
alex test

**Comment by alexanderrichards** [comment]
alex test

**Comment by alexanderrichards** [comment]
alex test

**Comment by egede** [comment]
retest this please

**Comment by alexanderrichards** [comment]
getting closer :wink: I see the FUSE mount now

**Comment by alexanderrichards** [comment]
alex test

**Comment by JinNamkung** [comment]
retest this please

**Comment by alexanderrichards** [comment]
alex test

**Comment by JinNamkung** [comment]
retest this please

**Comment by alexanderrichards** [comment]
alex test

**Comment by alexanderrichards** [comment]
@egede can we merge this now?

File: python/GangaLHCb/test/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
+FROM hepsw/cvmfs-lhcb:latest
+
+COPY ./ /root/ganga
+
+ENTRYPOINT . /etc/cvmfs/run-cvmfs.sh && \
+           . ~/.bashrc && \
+           export X509_CERT_DIR=/cvmfs/lhcb.cern.ch/etc/grid-security/certificates && \
+           export X509_VOMS_DIR=/cvmfs/lhcb.cern.ch/etc/grid-security/vomsdir && \
+           lhcb-proxy-init && \
```
Comment (rob-c) [inline python/GangaLHCb/test/Dockerfile]: don't you have to source the group login from cvmfs before you can do this or is this done through .bashrc?

Comment (alexanderrichards) [inline python/GangaLHCb/test/Dockerfile]: done through .bashrc

**Comment by alexanderrichards** [comment]
alex test

**Comment by alexanderrichards** [comment]
looks good 34/34 succeeded, I'll merge now

**Pull request merged**